packages/material-ui/src/Backdrop/Backdrop.test.js

```diff
@@ -1,9 +1,8 @@
 import * as React from 'react';
 import { expect } from 'chai';
 import { createMount, createClientRender, describeConformanceV5 } from 'test/utils';
-import Backdrop from './Backdrop';
+import Backdrop, { backdropClasses as classes } from './Backdrop';
```

Comment:
Best to test the public API

```suggestion
import Backdrop, { backdropClasses as classes } from '@material-ui/core/Backdrop';
```

Reply:
The same problem will be with Badge used as a template.

Reply:
True, we have many tests that don't follow this pattern, we migrate them as we work on these tests. It has become more intersting now, to make sure the classes are correctly exported in the barrel index file.